modules/fedimint-lnv2-common/src/gateway_api.rs

```diff
@@ -245,6 +247,16 @@ impl PaymentFee {
     }
 }
 
+impl Add for PaymentFee {
+    type Output = PaymentFee;
+    fn add(self, rhs: Self) -> Self::Output {
+        PaymentFee {
+            base: self.base + rhs.base,
+            parts_per_million: self.parts_per_million + rhs.parts_per_million,
+        }
+    }
+}
+
```

Comment:
Can this not be derived? If not, as its only used in one place please use it there directly instead of implementing a trait.

Reply:
It cannot be derived. It is used in multiple places (when returning the `RoutingInfo` and when checking if the newly configured fee is valid). Going to keep as is